### ISSUE TITLE
basil/oxford_asl --bolus parameter can accept a comma-separated list

### ISSUE THREAD

**Issue by andrewrosss** [PennLINC/aslprep#171]
Hi, I have a use-case related to the `oxford_asl` command (which the [`BASILCBF`]() interface wraps). I have some multi-delay ASL data which has the M0 image(s) included.

Since the data is multi-delay (and because the M0 is included) the `PostLabelingDelay` metadata field is an array of the form:

```text
"PostLabelingDelay": [0.0, 0.0, pld1, pld2, pld3, ...]
                      ^^^^^^^^
                      M0 images
```

As described in the BIDS spec for [`PostLabelingDelay`](https://bids-specification.readthedocs.io/en/v1.6.0/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#common-metadata-fields-applicable-to-both-pcasl-and-pasl) the M0 images appear as 0's (above).

Similarly, the `LabelingDuration` is also an array where we play the same game with the M0 Images, i.e. an array of the form:
```text
"LabelingDuration": [0.0, 0.0, ld1, ld2, ld3, ...]
                     ^^^^^^^^
                     M0 images
```

It turns out that the `--bolus` parameter for the `oxford_asl` command can be either a number or a list of numbers.

**tl;dr: I was wondering if the `bolus` parameter of the [`BASILCBF`]() interface could/should be made to accept either a `float` or `list[float]`.**

**Comment by a3sha2** [comment]
hi @andrewrosss  yes the bolus always take decimal 
https://github.com/PennLINC/aslprep/blob/master/aslprep/interfaces/cbf_computation.py#L727

the `tis` is converted to list before computation of BASIL which, it can take multiple pld 
https://github.com/PennLINC/aslprep/blob/master/aslprep/interfaces/cbf_computation.py#L724

remember you can only have one bolus duration. look at this slide 51 https://fsl.fmrib.ox.ac.uk/fslcourse/physics+apps/FSL_arterial_spin_labelling.pdf
